### PR TITLE
Add DreamOs64, northport and Osdev-Notes

### PR DESCRIPTION
**Pull request by dreamos82**
Adding:

Osdev-Notes, Dreamos64 and northport

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ It contains open source operating sytems and hobby operating systems as one can
 * [Brutal](https://brutal-org.netlify.app/), [GitHub](https://github.com/brutal-org/brutal) - An operating system inspired by brutalist design that combines the ideals of UNIX from the 1970s with modern technology and engineering
 * [Cyjon](https://github.com/CorruptedByCPU/Cyjon/) - pure x64 assembly language operating system (related to [Fern-Night](https://github.com/CorruptedByCPU/Fern-Night/) project).
 * [DragonOS](https://github.com/fslongjin/DragonOS) - An x86-64 OS build from scratch.
+* [Dreamos64](https://github.com/dreamos82/Dreamos64) - And x86-64 hobby os built from scratch
```
Comment (jubalh) [inline README.md:65]: `And` ?

Comment (dreamos82) [reply README.md:65]: oops... :)

**Comment by dreamos82** [comment]
should be fixed now!

**Comment by jubalh** [comment]
Great thanks! Can you squash the commits into one?

**Comment by dreamos82** [comment]
should be fixed now?

**Comment by jubalh** [comment]
Thanks!

**Pull request merged**